core/src/main/scala/kafka/admin/ConsumerGroupCommand.scala

```diff
@@ -83,12 +83,13 @@ object ConsumerGroupCommand extends Logging {
                 printAssignment(assignments, !opts.useOldConsumer)
               case Some("Stable") =>
                 printAssignment(assignments, !opts.useOldConsumer)
+              case None =>
```

Comment:
@harshach Sure, I assume you mean putting my response above in a code comment above. Thanks.